### DISCUSSION TITLE
Fonts! [Deadline: March 15]

### DISCUSSION THREAD

**Discussion by ebullient** [commonhaus/foundation#101] [Consensus Building]
[![🗳️ Vote progress](https://www.commonhaus.org/votes/commonhaus/foundation/101.svg)](https://github.com/commonhaus/foundation/discussions/101#discussioncomment-8751678 "DC_kwDOKRPTI84AhYo-")

### Proposal

Shorter turnaround on this one.

> [!IMPORTANT]
> View the details: [Round 5 fonts](https://docs.google.com/presentation/d/1ZbBFL1Qbs44pwvmzZboGdDHjAnHR7eqjykQK2_bLLxg/edit?usp=sharing)  

<img width="100%" alt="image" src="https://github.com/commonhaus/foundation-draft/assets/808713/aa68ac58-f7c1-462c-b1f2-ca36b33d541c">

### Voting group

@commonhaus/founders

### Consensus mechanism

**Rank them in order of preference.**

- (🚀, optional) Specific number if you have a favorite
- (👎, optional) Specific number if it is an absolute no

Constructive commentary is helpful.

[a comment by an automated account is not shown]

**Comment by aalmiray** [comment]
1 🚀 easier to read
2 👍 different but takes 1 extra nanosecond to read
3 👍 perhaps 1 or 2 pixels smaller?
4 👎
5 👎
6 👎
7 👎 

4 to 7 meesa no likey

**Comment by ebullient** [comment]
I'm with you, Andres. Same order, too.

**Comment by sigrunixia** [comment]
In order:
1 🚀
3 🚀
6 👍
2 👍
4 👎
5 👎
7 👎

**Comment by cowtowncoder** [comment]
In order:
7 🚀
1 👍
6👍
2
5
3
4

**Comment by yrodiere** [comment]
3 :rocket: 
1 :+1: 
2 :-1: 
4 :-1: 
6 :-1: 
7 :-1: 
5 :-1:

**Comment by kenfinnigan** [comment]
1 🚀
3
7
4
2
5 👎
6 👎

**Comment by gavinking** [comment]
4
2
1
3
5 👎
6 👎
7 👎

**Comment by nmcl** [comment]
1, 7, 3, 4, 5, 2, 6

**Comment by cealsair** [comment]
In order of preference from most preferred to least preferred:

- 7
- 2
- 6
- 5
- 4
- 1
- 3

**Comment by evanchooly** [comment]
7
6 ( just less heavy than the others )
1
3
2 💩   (moar kerning!)
4 💩 
5 💩 

All of them would "work," though.  But I will (not so?) silently judge choices of 2, 4, or 5.

**Comment by maxandersen** [comment]
2,3,1,7,6,5,4

**Comment by ebullient** [comment]
Quorum again! Well done, everyone, and thank you!
